### ISSUE TITLE
Be able to search loadouts by items in them

### ISSUE THREAD

**Issue by mannyrye** [DestinyItemManager/DIM#8571]
### Proposed change

Say that one wants to search for loadout that have the Austringer in them, or weapons with Demolitionist, or Iron Banner armor. Right now it can't be done, right?

### How does this fit into your workflow?

I can easily find the build that meets my needs, and then I can name the builds more poetically, without having to add parenthetical notes to the names.
Right now one can search by the contents of the notes, which is useful.

**Comment by bhollis** [comment]
Makes sense, though we'd probably want to introduce a search syntax for it, and then autocomplete, and then that'd be pretty messy to put in the loadout menu...

**Comment by Gix3612** [comment]
Commenting here a month later but, I'm pretty sure there's a workaround for this right now in the beta build. Go to the inventory page and find your item with a filter, such as `is:inloadout perkname:demolitionist` then use the triage tab to see what loadout it's in.
